src/contracts/protocol/tokenization/RwaAToken.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.10;
+
+import {IAccessControl} from 'src/contracts/dependencies/openzeppelin/contracts/IAccessControl.sol';
+import {IncentivizedERC20} from 'src/contracts/protocol/tokenization/base/IncentivizedERC20.sol';
+import {SafeCast} from 'src/contracts/dependencies/openzeppelin/contracts/SafeCast.sol';
+import {IERC20} from 'src/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
+import {Errors} from 'src/contracts/protocol/libraries/helpers/Errors.sol';
+import {AToken} from 'src/contracts/protocol/tokenization/AToken.sol';
+import {IRwaAToken} from 'src/contracts/interfaces/IRwaAToken.sol';
+import {IPool} from 'src/contracts/interfaces/IPool.sol';
+
+/**
+ * @title Aave ERC20 RWA AToken
+ * @author Aave
+ * @notice Implementation of the interest bearing token
+ * for Real-World Assets (RWAs) within the Aave protocol
+ */
```

Comment:
```suggestion
 * @title RwaAToken
 * @author Aave
 * @notice Implementation of the interest bearing token for Real-World Assets (RWAs)
 * @dev Functionalities are restricted to prevent aTokens from being transferred unless the action is performed by an authorized entity
 */
```